AdyenTwint/TwintComponent.swift

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import TwintSDK
+
+public final class TwintComponent {
+
+    private lazy var formViewController: FormViewController = {
+
+        return formViewController
+    }()
+
+    public var requiresModalPresentation: Bool = true
+
+    /// Initializes th
```

Comment:
Is this the whole file for the component? Seems empty :D
also missing documentation at the beginning.

Reply:
This is just a placeholder file. I am going to make changes to this in upcoming PR.